metatheory/plc-agda.nix

```diff
@@ -0,0 +1,16 @@
+{ mkDerivation, base, bytestring, cryptonite, ieee754
+, language-plutus-core, memory, stdenv, text
+}:
+mkDerivation {
+  pname = "plc-agda";
+  version = "0.1.0.0";
+  src = ./.;
+  isLibrary = false;
+  isExecutable = true;
+  executableHaskellDepends = [
+    base bytestring cryptonite ieee754 language-plutus-core memory text
+  ];
+  homepage = "https://github.com/input-output-hk/plutus-metatheory";
```

Comment:
shouldn't this be `homepage = "https://github.com/input-output-hk/plutus/metatheory"`?

Reply:
Ah yes, the cabal file is out of date, will fix.